dgl_sparse/src/elemenwise_op.cc

```diff
@@ -0,0 +1,33 @@
+/*!
+ *  Copyright (c) 2022 by Contributors
+ * \file sparse/elementwise_op.h
+ * \brief DGL C++ sparse elementwise operator implementation
+ */
+#include <sparse/elementwise_op.h>
+#include <sparse/sparse_matrix.h>
+#include <torch/custom_class.h>
+#include <torch/script.h>
+
+#include <memory>
+
+#include "./utils.h"
+
+namespace dgl {
+namespace sparse {
+
+c10::intrusive_ptr<SparseMatrix> SpSpAdd(
+    const c10::intrusive_ptr<SparseMatrix>& A,
+    const c10::intrusive_ptr<SparseMatrix>& B) {
```

Comment:
* Miss validity check, e.g., A and B should have the same shape, device, dtype, etc.
* Finish CSR/CSC implementation for adding two matrices of the same sparsity.
* Leave a TODO about adding two matrices of different sparsity. (also create an issue to track it).